### PR TITLE
Posts

### PR DESCRIPTION
**Pull request by feleck**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all.order('created_at DESC')
```
Comment (oskarlakner) [inline app/controllers/posts_controller.rb]: I'd put that order to a custom scope

File: app/views/posts/index.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+%h1 Posts
+- @posts.each do |post|
+  %h3= post.title
+  %p Created #{time_ago_in_words(post.created_at)} ago.
```
Comment (oskarlakner) [inline app/views/posts/index.haml:4]: such logic in views is a very bad practice. Later on I'll show you how to refactor it using decorators.

File: test/controllers/posts_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
```
Comment (oskarlakner) [inline test/controllers/posts_controller_test.rb]: please google how to turn off tests generators :). We'll be using rspec anyway

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (feleck) [reply app/controllers/posts_controller.rb]: scope added

File: test/controllers/posts_controller_test.rb (hunk 3 above)
Comment (feleck) [reply test/controllers/posts_controller_test.rb]: Additional generated files removed and modified generators

File: app/views/posts/index.haml (hunk 2 above)
Comment (feleck) [reply app/views/posts/index.haml:4]: As for now we're living it this way?

Comment (oskarlakner) [reply app/views/posts/index.haml:4]: yes

**Pull request merged**